tidynamics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "tidynamics" %}
+{% set version = "0.1.2.dev4" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "7b787ff9392c19194b483852a065cbdb4463a2f78958833b2b42a160d9730c6d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
```

Comment:
I don't think that `numpy` is required to build the package since it doesn't look to be imported in the `setup.py` either directly or as a side-effect.